test/unit/marked-spec.js

```diff
@@ -141,6 +153,29 @@ describe('use extension', () => {
     expect(html).toBe('<h1>heading</h1>\n');
   });
 
+  it('should call all walkTokens in reverse order', () => {
+    let walkedOnce = 0;
+    let walkedTwice = 0;
+    const extension1 = {
+      walkTokens(token) {
+        if (token.walkedOnce) {
+          walkedTwice++;
+        }
+      }
+    };
+    const extension2 = {
+      walkTokens(token) {
+        walkedOnce++;
+        token.walkedOnce = true;
+      }
+    };
+    marked.use(extension1);
+    marked.use(extension2);
+    marked('text');
+    expect(walkedOnce).toBe(2);
```

Comment:
walkedOnce is 2? Maybe it should be renamed?

Reply:
That variable counts how many tokens were walked at least once

Reply:
Perhaps `nWalkedOnce` ?